tools/dm/skip-replace-sqls.md

```diff
@@ -0,0 +1,591 @@
+            "worker": "127.0.0.1:8262",          # The IP:port (worker-id) of this DM-worker.
+            "msg": "",                           # The additional message for the failure to the error query in this DM-worker.
+            "subTaskError": [                    # The error information of the running subtask in this DM-worker.
+                {
+                    "name": "test",              # The task name.
+                    "stage": "Paused",           # The status of the current task.
+                    "unit": "Sync",              # The current processing unit of the running task.
+                    "sync": {                    # The error information of the binlog replication unit (sync).
+                        "errors": [              # The error information list of the current processing unit.
+                            {
+                                // The error information description.
+                                "msg": "exec sqls[[USE `db1`; ALTER TABLE `db1`.`tbl1` CHANGE COLUMN `c2` `c2` decimal(10,3);]] failed, err:Error 1105: unsupported modify column length 10 is less than origin 11",
+                                // The position of the failed binlog event.
+                                "failedBinlogPosition": "mysql-bin|000001.000003:34642",
+                                // The SQL statement that raises an error.
+                                "errorSQL": "[USE `db1`; ALTER TABLE `db1`.`tbl1` CHANGE COLUMN `c2` `c2` decimal(10,3);]"
+                            }
+                        ]
+                    }
+                }
+            ],
+            "RelayError": {                      # The error information of the relay processing unit in this DM-worker.
+                "msg": ""                        # The error information description.
+            }
+        }
+    ]
+}
+```
+
+#### sql-skip
+
+`sql-skip` allows you to preset a skip operation that is to be executed when the position or the SQL statement of the binlog event matches with the specified `binlog-pos` or `sql-pattern`. 
+
+##### Command usage
+
+```bash
+sql-skip <--worker=127.0.0.1:8262> [--binlog-pos=mysql-bin|000001.000003:3270] [--sql-pattern=~(?i)ALTER\s+TABLE\s+`db1`.`tbl1`\s+ADD\s+COLUMN\s+col1\s+INT] [--sharding] <task-name>
+```
+
+##### Arguments description
+
++ `worker`:
+    - Flag parameter, string, `--worker`
+    - If `--sharding` is not specified, `worker` is required; if `--sharding` is specified, `worker` is forbidden to use.
+    - `worker` specifies the DM-worker in which the presetted operation is going to be executed.
+
++ `binlog-pos`: 
+    - Flag parameter, string, `--binlog-pos`
+    - You must specify `binlog-pos` or `--sql-pattern`, and you must not specify both.
+    - If it is specified, the skip operation will be executed when `binlog-pos` matches with the position of the binlog event. The format is `binlog-filename:binlog-pos`, for example, `mysql-bin|000001.000003:3270`.
```

Comment:
Please limit the use of the word "will" and update similar expressions in this PR to present tense. Thanks!

Reply:
Okay :+1:

Reply:
Comment addressed. PTAL again. @lilin90